saleor/tests/e2e/promotions/test_promotions_query_with_different_parameters.py

```diff
@@ -0,0 +1,424 @@
+from datetime import datetime, timedelta
+
+import pytest
+from freezegun import freeze_time
+
+from ..metadata.utils import update_metadata, update_private_metadata
+from ..promotions.utils import create_promotion, promotions_query
+from ..sales.utils import create_sale
+from ..utils import assign_permissions
+
+# Should be able to query promotions with a different parameters CORE_2118
+
+
+# Step 1 - Promotions with the parameter: first: 10
+@pytest.mark.e2e
+def test_query_promotions_first_10_CORE_2118(
+    e2e_staff_api_client,
+    permission_manage_discounts,
+):
+    assign_permissions(e2e_staff_api_client, [permission_manage_discounts])
+
+    for i in range(10):
+        promotion_name = f"Promotion first {i + 1}"
+        create_promotion(e2e_staff_api_client, promotion_name)
+
+    promotions_list = promotions_query(e2e_staff_api_client, first=10)
+
+    assert len(promotions_list) == 10
+
+
+# Step 2 - Returns 10 promotions with CREATED_AT in descending order
+@pytest.mark.e2e
+def test_query_promotions_first_10_created_at_CORE_2118(
+    e2e_staff_api_client,
+    permission_manage_discounts,
+):
+    assign_permissions(e2e_staff_api_client, [permission_manage_discounts])
+
+    base_date = datetime(2023, 1, 1, 14, 1, 34, 61119)
+    created_at = [
+        (base_date + timedelta(days=i)).isoformat() + "+00:00" for i in range(10)
+    ]
+    with freeze_time(base_date):
+        for i in range(10):
+            promotion_name = f"Promotion first {i + 1}"
+            promotion = create_promotion(e2e_staff_api_client, promotion_name)
+            promotion["createdAt"] = created_at
+
+    promotions_list = promotions_query(e2e_staff_api_client, first=10)
```

Comment:
I know that by default we sort by DESC but for the purpose of this test can you add parameter `sort_by={"field": "CREATED_AT", "direction": "DESC"},`